firmware/main.c

```diff
@@ -709,11 +713,16 @@ static void cmd_motor_rt(BaseSequentialStream *chp, int argc, char *argv[]){
                                 start=false;
                                 chprintf(chp, "Pin 29 (Start) cleared! \r\n");
 			}
+			
                 break;
                 default: chprintf(chp, "Select one of the available modes \r\n");
 
 
         }
+//	palClearPad(pinPorts[pOpt1].gpio, pinPorts[pOpt1].pin); // Reset pins as its default state is high
```

Comment:
Personally, I really dislike  commented out code. Why's this commented out? If it's optional, it should be made like that, at compile time or at run time, whichever is preferred.